src/js/pagination/paging-collection.js

```diff
@@ -98,6 +106,35 @@
             },
 
             /**
+             * Returns the totalPages of the collection based on
+             * totalRecords and PageSize
+             *
+             * @returns {integer} The current page number.
+             */
+            getTotalPages: function () {
+                return this.state.totalPages;
+            },
+
+            /**
+             * Returns the total number of records the collection has
+             * on server
```

Comment:
Would you mind removing "on server"? It is assumed that the `PagingCollection` speaks to a server.